src/internal/util.go

```diff
@@ -2,6 +2,8 @@ package internal
 
 import "strings"
 
+// TODO: Move this somewhere to it's own package, not in internal.
+
```

Comment:
I agree, but let's move it later on. Honestly I think this can just be replaced with a direct call to strings.TrimPrefix. I'll put in another PR later to do that.